doc/RELEASE-NOTES.md

```diff
@@ -23,7 +23,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * *Describe architecture/environment changes here.*
 
 ### Bugfix
-* *Describe bugfix here.*
+* [OSDEV-673](https://opensupplyhub.atlassian.net/browse/OSDEV-673) found and fixed “TypeError: the JSON object must be str, bytes or bytearray, not list“
```

Comment:
Could you please briefly describe what is the problem you solved?  What has changed when you fixed this bug?
For instance, before the fix a user couldn't download the list, but after applying the fix they can.

Reply:
added description